setup.py

```diff
@@ -73,8 +73,9 @@ def find_version(*pathnames):
     license="BSD",
     name="CellProfiler",
     package_data={
-        "images": glob.glob(os.path.join("data", "**", "*"))
+        "cellprofiler": ['data/*']
```

Comment:
I ran `pip install .` in the root project directory and I got the following error when I ran `cellprofiler`:

```
$ cellprofiler
Traceback (most recent call last):
  File "/usr/local/bin/cellprofiler", line 11, in <module>
    load_entry_point('CellProfiler==3.0.0', 'console_scripts', 'cellprofiler')()
  File "/usr/local/lib/python2.7/site-packages/cellprofiler/__main__.py", line 152, in main
    app = cellprofiler.gui.app.App(0, workspace_path=workspace_path, pipeline_path=pipeline_path)
  File "/usr/local/lib/python2.7/site-packages/cellprofiler/gui/app.py", line 55, in __init__
    super(App, self).__init__(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/wx-3.0-osx_cocoa/wx/_core.py", line 8628, in __init__
    self._BootstrapApp()
  File "/usr/local/lib/python2.7/site-packages/wx-3.0-osx_cocoa/wx/_core.py", line 8196, in _BootstrapApp
    return _core_.PyApp__BootstrapApp(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/cellprofiler/gui/app.py", line 58, in OnInit
    import cellprofiler.gui.cpframe
  File "/usr/local/lib/python2.7/site-packages/cellprofiler/gui/cpframe.py", line 7, in <module>
    import cellprofiler.gui.figure
  File "/usr/local/lib/python2.7/site-packages/cellprofiler/gui/figure.py", line 41, in <module>
    import cellprofiler.gui.help.content
  File "/usr/local/lib/python2.7/site-packages/cellprofiler/gui/help/content.py", line 88, in <module>
    ("Using The Display Window Menu Bar", read_content("display_menu_bar.rst")),
  File "/usr/local/lib/python2.7/site-packages/cellprofiler/gui/help/content.py", line 22, in read_content
    with open(resource_filename, "r") as f:
IOError: [Errno 2] No such file or directory: '/usr/local/lib/python2.7/site-packages/cellprofiler/data/help/display_menu_bar.rst'
```

Changing the glob pattern to `'data/**/*'` appears to work.

Reply:
I'd like to add that my setup here is pure pip (no conda).

Reply:
Hmm, this was the exact issue I was trying to rectify. I was building a wheel and then performing the pip install on that. 
Interesting too, it appears to work fine for me without the glob: 
```
(cp-env) mattb@AI-Linux-mattb:~/git/CellProfiler$ cellprofiler
GLib-GIO-Message: Using the 'memory' GSettings backend.  Your settings will not be saved or shared with other applications.

(cellprofiler:16721): Gdk-WARNING **: gdk_window_set_icon_list: icons too large

(cellprofiler:16721): Gdk-WARNING **: gdk_window_set_icon_list: icons too large
Exiting the pipeline validation thread
(cp-env) mattb@AI-Linux-mattb:~/git/CellProfiler$ 
```

Reply:
Your suggested glob appears to work in a fresh environment with both `pip install .` and building from a wheel, I've added that change - thanks!

Reply:
Great! 

It seems odd that explicitly using `/` works over `os.path.join`, but the [documentation](http://setuptools.readthedocs.io/en/latest/setuptools.html#including-data-files) does specify:

> Also notice that if you use paths, you must use a forward slash (/) as the path separator, even if you are on Windows. Setuptools automatically converts slashes to appropriate platform-specific separators at build time.

Reply:
I have to agree it is odd, and even finding relevant documentation was difficult for this issue...it took me longer than I'd like to admit to find that the `include_package_data` flag was necessary :)